src/Epic/placeholders.ts

```diff
@@ -0,0 +1,30 @@
+import { getCountryName, getLocalCurrencySymbol } from './geolocation';
+
+const ALLOWED_PLACEHOLDERS = ['CURRENCY_SYMBOL', 'COUNTRY_NAME'];
+
+// we have to treat %%ARTICLE_COUNT%% placeholders specially as they are replaced
+// with react components, not a simple text substitution
+export const replaceNonArticleCountPlaceholders = (
+    content: string | undefined,
+    countryCode?: string,
+): string => {
+    if (!content) {
+        return '';
+    }
+
+    content = content.replace(/%%CURRENCY_SYMBOL%%/g, getLocalCurrencySymbol(countryCode));
+
+    const countryName = getCountryName(countryCode) ?? '';
+    content = countryName ? content.replace(/%%COUNTRY_NAME%%/g, countryName) : content;
```

Comment:
There is a scenario here where `%%COUNTRY_NAME%%` won't get replaced (if `countryName` isn't truthy). I don't think we'd want to render the epic if it hasn't been replaced. I wonder if we should remove `%%COUNTRY_NAME%%` from the list of supported placeholders since I don't _think_ it's used currently and it would allow us to sidestep the issue. The alternative would be to come up with a default countryName` and I'm not sure there's one which makes sense. What do you think?